docker-compose.yml

```diff
@@ -28,13 +28,21 @@ services:
       - mongodb
 
   scout-web:
-    build: .
-    container_name: scout-web
+    build:
+      context: .
+      dockerfile: Dockerfile-server
     expose:
-      - '5000'
+      - '8000'
     ports:
-      - '5000:5000'
-    command: scout --host mongodb --demo serve --host 0.0.0.0
+      - '8000:8000'
+    environment:
+      SCOUT_CONFIG: /home/worker/app/scout/server/config.py
```

Comment:
This is the env var that should point tp the Scout config file

Reply:
Still need the yaml as well for the CLI I guess, though not really the info in it anymore. But that's for laterz.